smoketests/tests/permissions.py

```diff
@@ -21,28 +19,22 @@ def test_call(self):
     def test_delete(self):
         """Ensure that you cannot delete a database that you do not own"""
 
-        self.new_identity()
-
         self.publish_module()
 
-        self.reset_config()
+        self.new_identity()
         with self.assertRaises(Exception):
             self.spacetime("delete", self.database_identity)
 
     def test_describe(self):
         """Ensure that anyone can describe any database"""
 
-        self.new_identity()
         self.publish_module()
 
-        self.reset_config()
-        self.new_identity()
-        self.spacetime("describe", "--json", self.database_identity)
+        self.spacetime("describe", "--anonymous", "--json", self.database_identity)
 
     def test_logs(self):
         """Ensure that we are not able to view the logs of a module that we don't have permission to view"""
 
-        self.new_identity()
```

Comment:
I don't believe this `new_identity` really added anything to the test (because we just use `new_identity` to change identities below anyway), but it hamstrung the test when running against servers that required a `spacetime login` to publish.